package.json

```diff
@@ -54,7 +54,7 @@
     "@0x/utils": "^4.5.2",
     "@balancer-labs/balancer-maths": "^0.0.19",
     "@balancer-labs/sor": "4.1.1-beta.4",
-    "@bgd-labs/aave-address-book": "2.21.1",
+    "@bgd-labs/aave-address-book": "4.7.0",
```

Comment:
Are all integrations that use aaave-address-book work ok after package bump?

Reply:
they should - it's still the same addresses (just some are split in different libraries)

Reply:
@JackieJoo I checked e2e tests - seemsall good. Also checked test env - it also looks good.